app/src/main/java/ua/syt0r/kanji/presentation/preview/screen/WritingPracticeScreenPreview.kt

```diff
@@ -25,7 +25,7 @@ import kotlin.time.Duration.Companion.seconds
 @Composable
 private fun KanjiPreview(
     darkTheme: Boolean = false,
-    isStudyMode: Boolean = true
+    isStudyMode: Boolean = false
```

Comment:
(didn't check if this is ever used without parameter specified, but default value probably shouldn't change here? or if it does that looks like it'd belong to another commit)

Reply:
Maybe this was originally `false` and changed for debugging, or just mistake. Because there's a `KanjiStudyPreview` function which explicitly passes `isStudyMode` as true.